### ISSUE TITLE
[AcidIsland] issue with world loading?

### ISSUE THREAD

**Issue by TheIronMinerLv** [tastybento/ASkyBlock-Bugs-N-Features#729]
Thank you for filing a bug report. Please complete these sections to help speed resolution.

**Server version? Remember to say if it is Spigot/Bukkit/Paper etc.**
PaperSpigot 1.12.2
**Plugin version:**
AcidIsland V3.0.8.6
**Is this a new install or upgrade from earlier plugin version? If upgrade, what version did you use before?**
I deleted acidisland folder and world folders too, but that didn't help at all. Let's say a new install.
**(Optional) What other plugins are you using now? (do /plugins and paste text from the console log here)**
AAC, AcidIsland, AdvancedOreGen, AdvancementsAddon, Arconix, BlockParticles, boosCooldowns, Boss, BungeeTabListPlus, ChatControl, ChatItem, ChatReaction, ChestShop, ConditionalCommands, Confiscate, CrazyAuctions, CrazyCrates, CustomPayloadFixer, DeluxeChat, DeluxeMenus, EasyBackup, EnchantsPlus, EnchLimiter, EpicSpawners, Essentials, EssentialsChat, EssentialsSpawn, FastAsyncWorldEdit, FeatherBoard, HolographicDisplays, JukeBox, KoTH, LeaderHeads, LibsDisguises, LiteBans, Lockette, MessageFilter, MVdWPlaceholderAPI, NametagEdit, OldCombatMechanics, PermissionsEx, PlaceholderAPI, ProQuests, ProtocolLib, ProtocolSupport, ProtocolSupportStuff, React, redclockdetector, SellWands, ShopGUIPlus, Skellett, SkinsRestorer, SkQuery, skRayFall, Skript, skript-db, skript-mirror, skript-yaml, skUtilities, StackMob, TradeMe, TuSKe, UUIDSpoof-Fix, Vault, VoidSpawn, VoxelSniper, WildSkript, WolvSK, WorldEdit, WorldGuard
**Description of issue. What happened?**
Basically, the plugin doesn't work properly, as it generated the world folders and even after restarts I get such error messages: https://pastebin.com/GMMXBwdb

Commands like /acid reload threw an internal error: https://pastebin.com/0S3AG9WL
**Steps to make this happen**
Fresh install the plugin on PaperSpigot 1.12.2, maybe something is wrong with my server itself, not quite sure.
**What do you think should happen?**
Please fix as I cannot continue to prepare my server for the public.

**Comment by TheIronMinerLv** [comment]
I updated my paperspigot and it worked.
